packages/sns/src/governance.canister.spec.ts

```diff
@@ -204,6 +206,43 @@ describe("Governance canister", () => {
     });
   });
 
+  describe("getProposal", () => {
+    it("should return the proposal", async () => {
+      const service = mock<ActorSubclass<SnsGovernanceService>>();
+      service.get_proposal.mockResolvedValue({
+        result: [{ Proposal: proposalMock }],
+      });
+
+      const canister = SnsGovernanceCanister.create({
+        canisterId: rootCanisterIdMock,
+        certifiedServiceOverride: service,
+      });
+      const res = await canister.getProposal({
+        proposalId: proposalIdMock,
+        certified: true,
+      });
+      expect(res).toEqual(proposalMock);
```

Comment:
shall we add an expect that it calls the correct endpoint?

Reply:
Thanks, done.